go/vt/vtgate/evalengine/comparisons_test.go

```diff
@@ -894,3 +893,79 @@ func TestCompareStrings(t *testing.T) {
 		})
 	}
 }
+
+// TestInOp tests the In operator comparisons
+func TestInOp(t *testing.T) {
+	tests := []testCase{
+		{
+			name: "integer In tuple",
+			v1:   NewLiteralInt(52), v2: Tuple{NewLiteralInt(52), NewLiteralInt(54)},
+			out: &T,
+			op:  &InOp{},
```

Comment:
nit: add cases for 
1. single value 
2. only null value
3. no value

in tuple

Reply:
Fixed